exception ValueError(int value) {
    message: string = "Wrong " + value to string + " - should be higher than 0: ";
}

$
Block
comment
$

bool is_even(int number){
    return number % 2 == 0;
}

void print_even_if_not_divisible_by_5(int number){
    while (number > 0) {
        # comment
        if (number % 5 == 0) {
            break;
        }elif (is_even(number)) {
            continue;
        }else{
            print("x: ", number);
            number = number - 1;
        }
    }
}
void main(){
    try{
        x = input() to int;
        if (x <= 0){
            throw ValueError(x);
        }
        print_even_if_not_divisible_by_5(x);
    }catch(BaseException e){
        print("Error: ", e.message, "\\n \t Value=", e.value, e.line, "\\n");
    }
}